Cargo.toml

```
@@ -3,4 +3,24 @@ name = "vex-prover"
 version = "0.1.0"
 edition = "2021"
 
+[profile.release]
+debug = 1
+
 [dependencies]
+stwo-prover = { git = "https://github.com/starkware-libs/stwo", rev = "a194fad", features = ["parallel"] }
+stwo-air-utils = { git = "https://github.com/starkware-libs/stwo", rev = "a194fad" }
+tracing = "0.1"
+itertools = "0.12.0"
+tracing-subscriber = "0.3"
+num-traits = "0.2.19"
+rayon = { version = "1.10.0" }
+
+[dev-dependencies]
+rand = "0.8.5"
+criterion = { version = "0.5", features = ["html_reports"] }
+vex-matching-engine = { git = "ssh://git@github.com/aerius-labs/VEX-matching-engine.git" }
```

Using SSH Git URLs in Cargo.toml might cause issues in CI/CD environments. Consider using HTTPS URLs with appropriate authentication.
```suggestion
vex-matching-engine = { git = "https://github.com/aerius-labs/VEX-matching-engine.git" }
```